internal/app/synchronizer/synchronizer_service_test.go

```diff
@@ -215,6 +234,7 @@ func runEvaluationTest(t *testing.T, tc *testData) {
 	for _, c := range tc.testCalls {
 		c := c
 		go func() {
+			ctx := context.Background()
```

Comment:
It's not clear because of how github does diffing but if you have the RpcTestContext or whatever it's called in scope use the `Context()` method.

Reply:
This is a unit test - so not really initializing any RPC server here.